### ISSUE TITLE
Add testing

### ISSUE THREAD

**Issue by 9gustin** [9gustin/react-notion-render#4]
Now the components of this package have no tests, they should have

**Comment by mrzmyr** [comment]
Hi @9gustin, do you already have any plans on this? 

Questions:
- Which framework should be used? ([react-testing-library](https://github.com/testing-library/react-testing-library))
- How does the directory structure look like (mock data, tests, coverage results)?
- Are there any GitHub Actions / Bots that can be leveraged on PRs?

**Comment by 9gustin** [comment]
Yes, i plan to add testing with react-testing-library. I know that library so i dont have problem with that. 

The problem here was that i'm started this package with [create-react-library](https://www.npmjs.com/package/create-react-library) and never can use the test command. Looks like if exist a conflict with typescript config and testing config.

If you run `npm run test`  the command runs, but this make changes in typescript config and the app doesnt build anymore(running npm run build). And test command doesnt run without this changes.

I've tried in a time but not found a solution, if you found it i would love you hahaha 😄  

In first the idea it's to add unit-testing to the components/hocs/utils.
